### ISSUE TITLE
DarkMode selector doesn't seem to be working

### ISSUE THREAD

**Issue by ct5845** [JavierM42/tailwind-material-colors#8]
Whatever i try the plugin seems to be only default to `@media (prefers-color-scheme: dark)`

````
const config = {
  darkMode: ["selector", "[data-theme=\"dark\"]"],

module.exports = withMaterialColors(
  config,
  {
    primary: "#00491e"
  },
  {
    scheme: "content"
  }
);
````

Stuck in "dark mode" as that's my local preference, can see in the variable outputs all the dark mode srtuff is being put under 
`@media (prefers-color-scheme: dark) :root {` and not my preferred selector. From what i can see of the source, looks like if it's not "class" it doesn't respect the selector.

Maybe this is using an older version of the `tailwind-mode-aware-colors`? seems to be pinned to v1 and looks like you've updarted to v2, perhaps fixing this there?

(ps. thanks for the plugin, does look like it's going to be immensely helpful!)

**Comment by JavierM42** [comment]
Thanks for the heads up!

---

_Why it was broken: feel free to skip this part_

This was a mistake on my end. Some time ago I approved a PR for `tailwind-mode-aware-colors` that added type definitions, but also inadvertently broke the ability for the module to be imported instead of required.

Thus, I hadn't been able to upgrade this package to use v2 of `tailwind-mode-aware-colors`, and v1 did not support the `selector` strategy (it didn't exist back then).

---

I've found the problem and released v3.0.2 of `tailwind-material-colors` which uses v2 of mode-aware-colors and should work with the `selector` strategy. Please let me know if this solves the issue 😁 

(Glad you find it helpful, that's always nice to hear!)

**Comment by ct5845** [comment]
Updated, works! brilliant stuff thank you.
